### DISCUSSION TITLE
E' possibile inserire dei link all'interno delle configurazioni testuali del software?

### DISCUSSION THREAD

**Discussion by Cangio84** [globaleaks/globaleaks-whistleblowing-software#3667] [Q&A]
Buongiorno,
è possibile inserire nel modello di domanda "casella di selezione" un opzione per poter allegare o inserire un link web cliccabile?
In alternativa è possibile migliorare il modello di domanda "termini di servizio". Qui è possibile inserire come allegato un link web cliccabile, ma viene proposto un campo non compilabile.

In allegato alcune schermate.

Grazie,

![modello-domanda_termini di servizio](https://github.com/globaleaks/GlobaLeaks/assets/144811052/43f6aa6c-02cc-4d25-ad08-573bec3fec9d)
![modello-domanda_casella di selezione](https://github.com/globaleaks/GlobaLeaks/assets/144811052/f79149e8-b809-49f8-b127-1cc0c653521a)

**Comment by evilaliv3** [comment]
Grazie per la domanda @Cangio84 

La maggior parte delle configurazioni testuali del software permette di definire degli hyperlink tramite linguaggio markdown.

E' sufficiente scrivere ad esempio ```[titolo](link)``` per ottenere un link con titolo specificato che punti all'indirizzo specificato.

**Comment by Cangio84** [comment]
Ciao @evilaliv3 ,
grazie della risposta.
Ho provato ad inserire nel nome del campo il codice come da te suggerito, ma non viene formattato. rimane il codice. La casella in questione è di tipo "selezione".
Grazie,
Cangio84

**Comment by cienne1** [comment]
Anche noi abbiamo una esigenza simile. Vorremmo inserire  una casella di spunta di presa visione dell'informativa privacy. L'ideale sarebbe poterla prevedere nel popup di accesso alla piattaforma, ma questo non mi risulta previsto.
In alternativa potete aiutarmi a capire come inserire una casella di spunta alla fine delle domande in cui semplicemente scrivere "ho preco visione dell'informativa privacy" e far cliccare (azione obbligatoria per poter inviare la segnalazione)?

**Reply by maelaborazioni** [reply]
si può far comparire dopo che l'utente si è loggato.
è in Utenti, tab Opzioni

**Reply by Cangio84** [reply]
Ciao maelaborazioni,
ringrazio per la risposta, ma il link all'informativa per la privacy deve essere disponibile per il segnalante non per gli utenti che gestiscono le segnalazioni. Ho provato l'impostazione che hai suggerito. Ben fatta, ma non è quello che ho richiesto. Il problema o meglio la richiesta è quella di poter inserire un link "cliccabile" su una domanda. La domanda che ho aggiunto ad un questionario personalizzato è una casella di tipo selezione e provando a "formattare" "scrivere" il link nel linguaggio markdown come @evilaliv3 non funziona, il link non diventa cliccabile
Grazie,
Alessandro

**Reply by maelaborazioni** [reply]
...e se provassi caricando un javascript?

![demo_js fw](https://github.com/globaleaks/GlobaLeaks/assets/35804199/fc0d47fe-82c0-4e9c-adb1-a796e9d8a57b)

**Reply by maelaborazioni** [reply]
peraltro se usassi insertAdjacentHTML anzichè inneHTML potresti andare in append alla domanda con il risultato di non doverla scrivere nel js ma sfuttando il meccanismo classico di GL

**Reply by evilaliv3** [reply]
Nel titolo delle domande non abbiamo previsto ne prevediamo al momento la possibilità di includere link ma lo stiamo già provando nelle "descrizioni" delle domande.

In merito invece alla necessità di esporre delle privacy policy, vi invitiamo ad usare la funzionalità specifica creando una domanda di tipo "Termini e condizioni" che vi permette di creare una classica textarea con dei termin, vi permette di definire il testo di accettazione e vi permette di esporre un link per scaricare la policy.

**Reply by maelaborazioni** [reply]
@evilaliv3 tipo così? (ho modificato un po' il css per renderlo più simile a una domanda "normale" - ho aperto un thread apposito sul discorso css)
![demo_ccs fw](https://github.com/globaleaks/GlobaLeaks/assets/35804199/931228aa-bebe-46d8-aa87-5b719ca80a3e)

**Reply by evilaliv3** [reply]
Non ho capito come questo si relazioni alla precedente domanda, ma si mi sembra vada bene!

**Reply by maelaborazioni** [reply]
col css ho tolto il checkbox e ed il riquadro con bordo e font monospace. Ma potrei voler usare questo trick solo per questa specifica domanda e non per tutte quelle di tipo "Termini di servizio"...

**Comment by Cangio84** [comment]
Ciao,
ringrazio entrambi per le risposte. @maelaborazioni: non ho capito da quale opzioni puoi caricare un css. Le schermate allegate dove si abilitano ?
@evilaliv3: avevo già provato ad utilizzare "termini di servizio" il problema che io voglio rendere obbligatorio la presa di visione dell'informativa privacy. Questa opzione è disponibile solo nel modello "casella di selezione" o scelta multipla. Ho provato anche ad inserire modello "gruppo di domande" con una prima "casella di selezione" e poi " termini di servizio", ma nelle condizioni di attivazione non ho come opzione la casella di selezione.
Ho provato anche ad inserire dopo una "casella di selezione" una "termini di servizio" con condizione di attivazione in base alla risposta della domanda precedete, purtroppo oltre ad essere tutto il testo visibile, il link non viene attivato in base alla risposta data precedentemente.
Spero di essermi spiegato.

Grazie,
Cangio84

**Reply by cienne1** [reply]
Se può essere utile, noi abbiamo risolto il problema prevedendo una domanda: hai visualizzato l'informativa privacy? risposta alternativa SI/NO. Prevedendo che se si seleziona NO non si può inviare la segnalazione. Così facendo, tra l'altro, la risposta resta salvata.

**Reply by maelaborazioni** [reply]
@Cangio84 Impostazioni > Allegati > css
